package.json

```diff
@@ -2,7 +2,7 @@
   "name": "ischemaview-phonegap-plugin-push",
   "description": "Register and receive push notifications. Forked to add functionality by iSchemaView",
   "types": "./types/index.d.ts",
-  "version": "3.2.0",
+  "version": "3.3.0-upandroid.1",
```

Comment:
```suggestion
  "version": "3.3.0",
```

Reply:
Is this a testing tag or are we already certain that this would be the final version for the update? If it's a testing branch we should probably leave the prerelease version on?

Reply:
I think we'll have issues using the pre-release version for cordova plugin? Ideally we don't want the plugin to fetch from github source

Reply:
Yeah I think this version is good to go per our recent testing.  And yes need to get a release tag so I can update the configs to not use github.

Reply:
It was my understanding we were going to merge this pre-release into master, then I would branch for the actual 3.3.0 release tag and submit a new PR.  Does that sound ok?  @shalase  @franco-octavianus

Reply:
I'm good with using a final version.

Reply:
Our normal process is to merge these `prerelease` changes into `master` -> branch off of `master` in a `tag/v3.0.0` branch -> create tag in `tag/v3.0.0` branch -> pr `tag/v3.0.0` into `master` -> push `v3.0.0` tag so it's built and deployed to nexus.

Reply:
Yes that is what @tyson-rapid  had told me and what I was going to do, so I could learn and follow the standard process. Another PR coming. Thanks!